frontend/src/views/CellGuide/components/CellGuideCard/style.ts

```diff
@@ -211,6 +204,7 @@ export const MobileTooltipWrapper = styled.div`
   display: flex;
   flex-direction: column;
   height: calc(100vh - ${HEADER_HEIGHT_PX}px);
+  overscroll-behavior: contain;
```

Comment:
this fixes a bug with the tooltip overlay where users could scroll past the bounds of the tooltip to scroll the container underneath